.circleci/config.yml

```diff
@@ -39,44 +39,40 @@ commands:
           command: |
             cd && touch $BASH_ENV
 
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              echo 'export INFRA_PATH=iac/conf/websites/prod/lunie.io' >> $BASH_ENV
-              echo 'export DEPLOYMENT_ENV=lunie.io' >> $BASH_ENV
-              echo 'export GIT_BRANCH=${CIRCLE_BRANCH}' >> $BASH_ENV
+            if [ "${CIRCLE_BRANCH}" = "master" ]; then
+              echo 'export DISTRIBUTION_ID=E3NNXI2PHNC362' >> $BASH_ENV
+              echo 'export DIST_ID_TESTNET=E27N83VZ9QZFET' >> $BASH_ENV
+
+              echo 'export BUCKET_NAME=lunie.io' >> $BASH_ENV
+              echo 'export BUCKET_NAME_TESTNET=test.lunie.io ' >> $BASH_ENV
```

Comment:
This will deploy the same build to `test.lunie.io`. I will merge anyway for now. Need to improve later.